core/src/main/scala/org/apache/spark/deploy/history/HistoryPage.scala

```diff
@@ -82,72 +65,37 @@ private[history] class HistoryPage(parent: HistoryServer) extends WebUIPage("")
               <h4>
                 Showing {actualFirst + 1}-{last + 1} of {allAppsSize}
```

Comment:
does this stay in sync with what is actually showing in datatable?  It might be better to just remove this since data table also has the number showing at bottomt